api/client/contextdialer.go

```diff
@@ -192,6 +192,12 @@ func NewDialer(ctx context.Context, keepAlivePeriod, dialTimeout time.Duration,
 		// Base direct dialer.
 		var dialer ContextDialer = netDialer
 
+		// TODO(greedy52) currently there is no use case where both cfg.proxyHeaderGetter
```

Comment:
What to do here, though? Should it be just a comment?

Reply:
good call. will remove TODO.